src/components/PopUpForm.js

```diff
@@ -1,42 +1,62 @@
-import React from "react";
-import '../App.css';
+import React, { Component } from "react";
 
-const PopUpForm = (props) => {
-    const {priority, setPriority, date, setDate, description, setDescription,todosDetails, setTodosDetails, id}=props;
-    const inputDescriptionHandler=(e)=>{
-        setDescription(e.target.value);
-    }
-    const inputDateHandler=(e)=>{
-        setDate(e.target.value);
-    }
-    const inputPriorityHandler=(e)=>{
-        setPriority(e.target.value);
-    }
-    const submitDetailsHandler=(e)=>{
-        e.preventDefault();
-        console.log(todosDetails);
-         setTodosDetails([...todosDetails, {description: description, date:date, priority:priority  }]);
-    }
-    
+class PopUpForm extends Component {
+  constructor() {
+    super();
+    this.state = {
+      description: "",
+      date: "",
+      priority: "",
+    };
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleChange(event) {
+    const { name, value } = event.target;
+    this.setState({
+      [name]: value,
+    });
+  }
+
+  render() {
     return (
+      <main className="todo-popup-container">
+        <form className="popup-form">
+          <input
+            name="description"
+            value={this.state.description}
```

Comment:
destructure before using
```suggestion
            value={description}
```